engines/griffon/cutscenes.cpp

```diff
@@ -182,6 +186,21 @@ void GriffonEngine::showLogos() {
 }
 
 void GriffonEngine::intro() {
+	#ifdef USE_TTS
+		Common::TextToSpeechManager *_ttsMan = g_system->getTextToSpeechManager();
+
+		Common::String result;
+
+		result+=story[0];
+
+		for(int i=10; i< ARRAYSIZE(story); ++i){
+			result+=story[i];
+		}
+
+		if (_ttsMan != nullptr)	
```

Comment:
The whole code should be guarded by that null check - the constructed string is not needed if `ttsMan == nullptr`